openshift.yaml

```diff
@@ -266,6 +274,24 @@ parameters:
     hidden: true
     default: ''
 
+  # Red Hat satellite subscription parameters
+  sat6_hostname:
+    type: string
+    description: >
+      The hostname of the Satellite 6 server which will provide software updates
+    default: ''
+
+  sat6_organization:
+    type: string
+    description: >
+      An organization string provided by Sat6 to group subscriptions
+    default: ''
+
+  sat6_activationkey:
+    type: string
+    description: >
+      An activation key string provided by Sat6 to enable subscriptions
```

Comment:
There is missing default value here so stack-create fails when using an existing env file.